chainer/links/connection/deconvolution_2d.py

```diff
@@ -66,23 +68,17 @@ def __init__(self, in_channels, out_channels, ksize, stride=1, pad=0,
         W_shape = (in_channels, out_channels, kh, kw)
         super(Deconvolution2D, self).__init__(W=W_shape)
 
-        if initialW is not None:
-            assert initialW.shape == \
-                (in_channels, out_channels, kh, kw)
-            self.W.data[...] = initialW
-        else:
-            self.W.data = numpy.random.normal(
-                0, wscale * math.sqrt(1. / (kh * kw * in_channels)),
-                (in_channels, out_channels, kh, kw)
-            ).astype(numpy.float32)
+        if hasattr(initialW, 'shape'):
+            assert initialW.shape == (in_channels, out_channels, kh, kw)
+        initializers.init_weight(self.W.data, initialW, scale=wscale)
 
-        if initial_bias is not None:
-            assert initial_bias.shape == (out_channels,)
-            self.add_param('b', out_channels)
-            self.b.data[...] = initial_bias
-        elif not nobias:
+        if not nobias:
```

Comment:
I found it is a matter independent of initializers.

I would like to share the situation to avoid confusion. This is the problem of priority of `nobias` and `initial_bias`.
`Linear` and `Convolution2D` give priority to `nobias` (i.e. when `nobias` is `True`, `self.b` is `None` regardless of `initial_bias`). On the other hand, current `Deconvolution2D` first consider `initial_bias` (i.e. `self.b` does have a value when `initial_bias` is not `None`). I fixed the priority of `Deconvolution2D` and align it with `Linear` and `Convolution2D` in this PR.

There are two choices:
* To keep the backward compatibility, the orders of priority of these links should be kept.
* To regard this difference as a bug of `Deconvolution2D` (or `Linear` and `Convolution2D`) and fix it (in this PR or as a another PR).

I think these three Links should treat `initial_bias` and `nobias` in the same way. So I recommend the latter one. I would like to know your opinion.